src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -135,6 +139,11 @@ public class ScriptHost : JobHost, IScriptJobHost
             _primaryHostStateProvider = primaryHostStateProvider;
             _bindingProviders = new List<IScriptBindingProvider>(bindingProviders);
             _metadataProvider = metadataProvider;
+            _eventSubscriptions.Add(EventManager.OfType<WorkerProcessErrorEvent>()
```

Comment:
Can we move this logic to the `LanguageWorkerChannelManager` or introduce a service that manages (and initializes) the worker. That would also allow us to move the language worker initialization code at the bottom out of the ScriptHost and remove the additional responsibility that doesn't fit here.

Reply:
This is specifically to populate FunctionErrors for each function if something goes wrong with the language worker process.